src/DynamoUtilities/Utils.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace DynamoUtilities
+{
+    public class Utils
```

Comment:
This should be a static class, but more importantly, classes called "Utils" often set off alarm bells in my head.  This class has a very specific purpose.  It should have a name like `AsmPathUtilities`.